docs/sources/installation/sizing/index.md

```diff
@@ -86,7 +103,8 @@ This tool helps to generate a Helm Charts `values.yaml` file based on specified
 </style>
 
 <script>
-const API_URL = `https://logql-analyzer.grafana.net/next/api/sizing`
+//const API_URL = `https://logql-analyzer.grafana.net/next/api/sizing`
```

Comment:
don't think you want to commit the localhost url

Reply:
Yes. Thanks for reminding me. This is for testing.